yarn-project/aztec-sandbox/src/sandbox.ts

```diff
@@ -65,12 +82,38 @@ export async function createSandbox(config: Partial<SandboxConfig> = {}) {
   const hdAccount = mnemonicToAccount(config.l1Mnemonic ?? MNEMONIC);
   const privKey = hdAccount.getHdKey().privateKey;
 
-  const l1Contracts = await waitThenDeploy(aztecNodeConfig.rpcUrl, hdAccount);
+  const l1Artifacts: L1ContractArtifactsForDeployment = {
+    contractDeploymentEmitter: {
+      contractAbi: ContractDeploymentEmitterAbi,
+      contractBytecode: ContractDeploymentEmitterBytecode,
+    },
+    registry: {
+      contractAbi: RegistryAbi,
+      contractBytecode: RegistryBytecode,
+    },
+    inbox: {
+      contractAbi: InboxAbi,
+      contractBytecode: InboxBytecode,
+    },
+    outbox: {
+      contractAbi: OutboxAbi,
+      contractBytecode: OutboxBytecode,
+    },
+    rollup: {
+      contractAbi: RollupAbi,
+      contractBytecode: RollupBytecode,
+    },
+  };
```

Comment:
For another PR, but I'd like to go back to having a single place where we define this.

Reply:
Agreed